### ISSUE TITLE
Accept a custom chart id

### ISSUE THREAD

**Issue by kgodsmark** [xr0master/chartjs-react#14]
I'm migrating over from react-chartjs-2. Would it be possible to accept an id provided to ReactChart to be used instead of the generated one here? https://github.com/xr0master/chartjs-react/blob/master/src/ReactChart.tsx#L38

E.g.
```
    <ReactChart
      id="custom-chart-id"
      type="bar"
      data={chartData}
      options={chartOption}
      height={400}
    />
```

**Comment by xr0master** [comment]
Hey. Thank you for your feedback!
The feature is implemented in 3.3.1

**Comment by kgodsmark** [comment]
Thanks so much for this - the migration from `react-chartjs-2` is complete and went well 👍
